core/src/main/java/tc/oc/pgm/teams/TeamMatchModule.java

```diff
@@ -538,7 +533,7 @@ public void balanceTeams() {
           .getLogger()
           .info(
               "Bumping a player from stacked team "
-                  + team.getColoredName()
+                  + team.getDisplayName()
```

Comment:
If this is the only reason we have `getDisplayName`, just remove the logging statements. TBH, we don't need them here.